packages/xo-server/src/api/xosan.js

```diff
@@ -861,7 +861,7 @@ async function umountDisk(localEndpoint, diskMountPoint) {
 }
 
 async function createNewDisk(xapi, sr, vm, diskSize) {
-  const vdiMax = 2093050 * Math.pow(2, 20)
+  const vdiMax = 2088960 * Math.pow(2, 20)
```

Comment:
Where does this value come from?

Reply:
```
Apr  6 13:57:49 host1-5xlgjv2 xapi: [error||55743 ||backtrace] Async.VDI.create R:3dfbade4c4fd failed with exception Server_error(SR_BACKEND_FAILURE_79, [ ; VDI Invalid size [opterr=VDI size must be between 1 MB and 2088960 MB];  ])
Apr  6 13:57:49 host1-5xlgjv2 xapi: [error||55743 ||backtrace] Raised Server_error(SR_BACKEND_FAILURE_79, [ ; VDI Invalid size [opterr=VDI size must be between 1 MB and 2088960 MB];  ])
```

Reply:
Please add a comment in the code to explain this :-)